src/xml_db/parse/entry.rs

```diff
@@ -42,7 +43,6 @@ impl FromXml for Entry {
                                 .split(|c| c == ';' || c == ',')
                                 .map(|x| x.to_owned())
                                 .collect();
-                            out.tags.sort();
```

Comment:
@sseemayer I ran into problems because of that line in the past as well. I agree that we should remove it.